applications/nrf5340_audio/src/modules/sd_card.c

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+
+/* Define _POSIX_C_SOURCE before including <string.h> in order to use `strtok_r`. */
+#define _POSIX_C_SOURCE 200809L
```

Comment:
In Zephyr side, some use `#undef` before to protect, if this is already defined.

I'm unsure of all the combinations, I just use default, so not sure if this is defined or not.

Reply:
Safest way, in case someone set this from cmake
```c
#ifdef _POSIX_C_SOURCE
#undef _POSIX_C_SOURCE
#endif
/* Define _POSIX_C_SOURCE before including <string.h> in order to use `strtok_r`. */
#define _POSIX_C_SOURCE 200809L
```

Reply:
Just for info: it is safe to undef a macro even if it was never defined. So the ifdef is not needed.